.github/workflows/windows.yml

```
@@ -169,6 +169,18 @@ jobs:
           sed -i "s|%GITHUB_REPO%|$GITHUB_REPOSITORY|"       dest/README.txt
           # Create dir for zipping
           mv dest dosbox-staging-windows-${{ env.VERSION }}
+
+      - name: Windows Defender AV Scan
+        shell: powershell
+        run: |
+          $ErrorActionPreference = 'Stop'
+          $dosboxDir = "./dosbox-staging-windows-${{ env.VERSION }}"
+          & 'C:\Program Files\Windows Defender\MpCmdRun.exe' -Scan -ScanType 3 -File $dosboxDir
```

```suggestion
          & 'C:\Program Files\Windows Defender\MpCmdRun.exe' -Scan -ScanType 3 -File $dosboxDir -DisableRemediation
```
After some testing, we need to add `-DisableRemediation` to make Windows Defender fail when virus detected, otherwise it will try to clean the virus automatically and return success. 

If not cleaning the virus is a concern, there a couple of ways I can parse out the logs and return an error without checking the return codes.